dataset_eval/gaia_wrapper.py

```diff
@@ -44,9 +46,29 @@ def train_model(project_dir, groundtruth_file, filelist_file):
         datasets_dir=os.path.join(project_dir, "datasets"),
         results_dir=os.path.join(project_dir, "results"),
     )
+    """
+    Converting the preferences from string type to list
+    """
+    c_value = [int(value) for value in c_value.split(',')]
```

Comment:
We need error checking here - if a person enters a string then the `int()` cast will fail. I think it's OK to silently skip non-numerical values here

Reply:
c_value is getting stored in the database as `string` type. for example: '1 2 3 4 5'
I am converting it to a list of integers similar to what is saved as default value in https://github.com/MTG/gaia/blob/1d593b89a65cbf0a6b3f1e70cb34e8d067dc6fd5/src/bindings/pygaia/scripts/classification/classification_project_template.yaml#L99